### PR TITLE
add umlet

### PR DESCRIPTION
**Pull request by rgl**
Add package for [UMLet](https://www.umlet.com/).

This should close https://github.com/chocolatey/chocolatey-package-requests/issues/466

### CODE REVIEW

File: automatic/umlet/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Free UML Tool for Fast UML Diagrams.
```
Comment (majkinetor) [inline automatic/umlet/README.md]: Readme has a standard form. Yoiu miss screenshot, features, link to package etc.

File: automatic/umlet/umlet.nuspec [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>umlet</id>
+    <version>14.3.0</version>
+    <title>Meld</title>
```
Comment (majkinetor) [inline automatic/umlet/umlet.nuspec]: Wrong title

File: automatic/umlet/umlet.nuspec [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <authors>umlet</authors>
+    <owners>chocolatey rgl</owners>
```
Comment (majkinetor) [inline automatic/umlet/umlet.nuspec]: comma between owners

File: automatic/umlet/umlet.nuspec [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    <licenseUrl>https://github.com/umlet/umlet/blob/master/LICENCE.txt</licenseUrl>
+    <projectUrl>https://www.umlet.com/</projectUrl>
+    <iconUrl>https://cdn.jsdelivr.net/gh/umlet/umlet@b89451cc1026f1dc42884c2b5b8d824bf770d8e6/umlet-standalone/src/exe/img/umlet_logo.png</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description><![CDATA[[UMLet](https://www.umlet.com/) is a free, open-source UML tool with a simple user interface: draw UML diagrams fast, build sequence and activity diagrams from plain text, export diagrams to eps, pdf, jpg, svg, and clipboard, share diagrams using Eclipse, and create new, custom UML elements. UMLet runs stand-alone or as Eclipse plug-in on Windows, OS X and Linux.
+]]></description>
+    <summary>Free UML Tool for Fast UML Diagrams.</summary>
+    <releaseNotes>https://www.umlet.com/changes.htm</releaseNotes>
+    <tags>foss cross-platform uml gui admin</tags>
```
Comment (majkinetor) [inline automatic/umlet/umlet.nuspec]: I suggest you add `documentation` tag.

Comment (rgl) [reply automatic/umlet/umlet.nuspec]: to where?

**Comment by rgl** [comment]
I've made the requested changes.

Please note that the screenshot url inside README.md and the nuspec need to be changed after the commit to point to the specific commit id, e.g. to `https://cdn.jsdelivr.net/gh/chocolatey/chocolatey-coreteampackages@<revision>/automatic/umlet/screenshot.png`

**Comment by majkinetor** [comment]
FYI, umlet run didn't succeed once merged. See report.

**Comment by majkinetor** [comment]
I fixed it.

**Comment by AdmiringWorm** [comment]
@rgl @majkinetor 
Please take care on how the License file is named.
It was failing the automated validation, because the validator didn't like the name `Licence`.
I've changed it now, and re-pushed the package (hopefully).

**Comment by majkinetor** [comment]
Its also one of my typical errors, no wonder I didn't notice it. Thx @AdmiringWorm

**Comment by rgl** [comment]
Oh, a typo. Sorry! Will try to get it right next time.

**Comment by AdmiringWorm** [comment]
I hadn't noticed it myself, which is why it's a good idea to keep an eye out on the chocolatey moderation emails from time to time.

That was the only reason I caught it.

**Pull request merged**